libs/react/src/lib/tabs/tabs.tsx

```diff
@@ -28,6 +34,7 @@ export const Tabs = ({ list, disabled, children, ...props }: TabsProps) => {
             role="tab"
             key={index}
             data-index-number={index}
+            data-disabled={value.disabled}
```

Comment:
Use aria-disabled instead and  omit href attribute when disabled. I've updated chlorophyll which now inherits behaviour from normal link, se updated storybook [here](https://sebgroup.github.io/green/pr-368/chlorophyll/?path=/docs/components-tabs--page#tabs-with-disabled-tab).